doc/userguide/src/CreatingTestData/AdvancedFeatures.rst

```diff
@@ -371,9 +371,9 @@ lower limit, upper limit and step.
 .. table:: For in range examples
    :class: example
 
-   ================  ===============  ===========  =========  ===========  ========  =======
-      Test Case          Action        Argument     Argument    Arg          Arg       Arg
-   ================  ===============  ===========  =========  ===========  ========  =======
+   ================  ===============  ===========  =========  ===========  ========  ========
+      Test Case          Action        Argument     Argument    Argument   Argument  Argument
+   ================  ===============  ===========  =========  ===========  ========  ========
```

Comment:
Unless you can figure out why Sphinx hates my tables, I'm all for switching to a plaintext format.